backend/seed.py

```
@@ -286,6 +287,48 @@ def add_user_goals():
     print("User goals added successfully.")
 
 
+def add_hints():
```

Ruff (0.8.2)

Missing return type annotation for public function `add_hints`

Add return type annotation: `None`

(ANN201)